git-open

```diff
@@ -167,7 +180,9 @@ case $( uname -s ) in
 esac
 
 # Allow printing the url if BROWSER=echo
-if [[ $BROWSER != "echo" ]]; then
+if [[ $BROWSER == "echo" ]]; then
+  openopt=''
```

Comment:
what's this about?

Reply:
It's to fix BATS testing on WSL, which still doesn't work after d97b7f36aadb219e91c17b1181b5f5ace8e9e15a. Here's the test output on my local machine on master:

```bash
[avalera:~/git-open-clean] master*
± npm test

> git-open@2.0.0 test /home/avalera/git-open-clean
> npm run unit && npm run lint:package && npm run lint:readme && npm run lint:editorconfig


> git-open@2.0.0 unit /home/avalera/git-open-clean
> bats test/

 ✓ test environment
 ✓ help text
 ✓ invalid option
 ✗ gh: basic
   (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 239,
    in test file test/git-open.bats, line 46)
     `assert_output "https://github.com/user/repo"' failed
   Switched to a new branch 'master'
   Reset branch 'master'

   -- output differs --
   expected : https://github.com/user/repo
   actual   : Start https://github.com/user/repo
   --

 ✗ gh: branch
   (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 239,
    in test file test/git-open.bats, line 53)
     `assert_output "https://github.com/user/repo/tree/mybranch"' failed
   Switched to a new branch 'master'
   Switched to a new branch 'mybranch'

   -- output differs --
   expected : https://github.com/user/repo/tree/mybranch
   actual   : Start https://github.com/user/repo/tree/mybranch
   --

...
```

The rest of it is basically the same thing, correct URL's on all tests but with the prefix `Start` from that `$openopt` variable not being unset until after the command is executed.